test-casskop-operator/test/atomic-1.yaml

```diff
@@ -0,0 +1,25 @@
+project: casskop-operator
```

Comment:
Could you replace the atomic config with the obs gap config that can detect this bug since we plan to use obs gap to detect this bug?

Reply:
Let me do this in a separate commit